.github/CONTRIBUTING.md

```diff
@@ -16,11 +16,17 @@ cd vercel
 corepack enable
 pnpm install
 pnpm build
-pnpm lint
-pnpm test-unit
 ```
 
-Make sure all the tests pass before making changes.
+To run the e2e tests, you'll need a `VERCEL_TOKEN`. To get that token, login to https://vercel.com, go to Settings, Tokens, then create a "Full Account" token to use for this purpose.
+
+Save that token or export it in your terminal profile.
+
+```
+export VERCEL_TOKEN="my-copied-token"
+```
+
+Then run the tests with `pnpm turbo run test-e2e`.
```

Comment:
We should also put a big warning here that this will create a ton of deployments and it might even get rated limited by hobby account limits if run in parallel.